### DISCUSSION TITLE
TypeError: Class extends value undefined is not a constructor or null

### DISCUSSION THREAD

**Discussion by cjdunteman** [ueberdosis/tiptap#3532] [General Questions] [answered]
I had TipTap working well in a React Vite project, have converted to Next.js and am running into this compile error. Not sure what is going on. I'm not using any class components, so not sure where this is happening.

![image](https://user-images.githubusercontent.com/37129024/207181853-016980b1-ecd1-4fbd-b493-2c495a646af1.png)

Full repo is public [here](https://github.com/cjdunteman/jot)

**Comment by somogyibence** [comment, marked as answer]
Hey there! I just ran into the same issue with a brand new Next.js project. Try adding the following line to the top of your `TipTap.tsx`: 

```
'use client';
```

This will allow the class components within tiptap to be rendered on the client-side.

**Reply by cjdunteman** [reply]
Thank you!
